packages/google_ml_kit/README.md

```diff
@@ -17,28 +17,28 @@ Go to each plugin listed bellow to read about their documentation and requiremen
+| [Face Mesh Detection (Beta)](https://developers.google.com/ml-kit/vision/face-mesh-detection)   | [google_mlkit_face_mesh_detection](https://pub.dev/packages/google_mlkit_face_mesh_detection) [![Pub Version](https://img.shields.io/pub/v/google_mlkit_face_mesh_detection)](https://pub.dev/packages/google_mlkit_face_mesh_detection)                 | ✅      | ❌  |
+| [Text Recognition v2](https://developers.google.com/ml-kit/vision/text-recognition/v2)          | [google_mlkit_text_recognition](https://pub.dev/packages/google_mlkit_text_recognition) [![Pub Version](https://img.shields.io/pub/v/google_mlkit_text_recognition)](https://pub.dev/packages/google_mlkit_text_recognition)                             | ✅      | ✅  |
+| [Image Labeling](https://developers.google.com/ml-kit/vision/image-labeling)                    | [google_mlkit_image_labeling](https://pub.dev/packages/google_mlkit_image_labeling) [![Pub Version](https://img.shields.io/pub/v/google_mlkit_image_labeling)](https://pub.dev/packages/google_mlkit_image_labeling)                                     | ✅      | ✅  |
+| [Object Detection and Tracking](https://developers.google.com/ml-kit/vision/object-detection)   | [google_mlkit_object_detection](https://pub.dev/packages/google_mlkit_object_detection) [![Pub Version](https://img.shields.io/pub/v/google_mlkit_object_detection)](https://pub.dev/packages/google_mlkit_object_detection)                             | ✅      | ✅  |
+| [Digital Ink Recognition](https://developers.google.com/ml-kit/vision/digital-ink-recognition)  | [google_mlkit_digital_ink_recognition](https://pub.dev/packages/google_mlkit_digital_ink_recognition) [![Pub Version](https://img.shields.io/pub/v/google_mlkit_digital_ink_recognition)](https://pub.dev/packages/google_mlkit_digital_ink_recognition) | ✅      | ✅  |
+| [Pose Detection (Beta)](https://developers.google.com/ml-kit/vision/pose-detection)             | [google_mlkit_pose_detection](https://pub.dev/packages/google_mlkit_pose_detection) [![Pub Version](https://img.shields.io/pub/v/google_mlkit_pose_detection)](https://pub.dev/packages/google_mlkit_pose_detection)                                     | ✅      | ✅  |
+| [Selfie Segmentation (Beta)](https://developers.google.com/ml-kit/vision/selfie-segmentation)   | [google_mlkit_selfie_segmentation](https://pub.dev/packages/google_mlkit_selfie_segmentation) [![Pub Version](https://img.shields.io/pub/v/google_mlkit_selfie_segmentation)](https://pub.dev/packages/google_mlkit_selfie_segmentation)                 | ✅      | ✅  |
+| [Subject Segmentation (Beta)](https://developers.google.com/ml-kit/vision/subject-segmentation) | [google_mlkit_subject_segemtation](https://pub.dev/packages/google_mlkit_subject_segemtation) [![Pub Version](https://img.shields.io/pub/v/google_mlkit_subject_segemtation)](https://pub.dev/packages/google_mlkit_subject_segemtation)                 | ❌      | ❌  |
```

Comment:
We are missing a check icon for android here @fbernaly

Reply:
done!